tests/specs/lemmas.k

```diff
@@ -104,6 +115,14 @@ module LEMMAS
 
     rule #asWord(WS) >>Int M => #asWord(WS [ 0 .. #sizeByteArray(WS) -Int (M /Int 8) ])  requires 0 <=Int M andBool M modInt 8 ==Int 0 [simplification]
 
+  // ########################
+  // Boolean Logic
+  // ########################
+
+    rule X:Int ==Int A:Int orBool X ==Int B:Int orBool X ==Int C:Int orBool X ==Int D:Int => X ==Int D
+         requires X =/=Int A andBool X =/=Int B andBool X =/=Int C
+         [simplification]
```

Comment:
This should likely be moved to a more specific file. I guess it's just for the MCD proofs, so should be moved to `tests/specs/mcd/verification.k`.

Reply:
Simply because I don't like this lemma very much, it's not very general or beautiful.

Reply:
Agreed.

Reply:
Unresolving this until the code shows up in the PR making the change.

Reply:
Should be there now.